_data/projects/validator/releases/9.0/series.yml

```diff
@@ -20,7 +20,7 @@ maven:
       summary: Set of test utilities that can help testing custom constraints.
 integration_constraints:
   java:
-    version: 17, 21 or 22
+    version: 17, 21, 22 or 23
```

Comment:
it's still in beta, same as ORM 7, so just that I know how to proceed ... should I remove the 22 testing from main (which is 9.0) or should I keep it and then ORM 7.0 mention the 22 🙈

Reply:
> should I remove the 22 testing from main (which is 9.0)

Yes:

* Upon versions of OpenJDK reaching EA, we should update the main branch of a project to start testing against it.
* Upon versions of OpenJDK reaching GA, we should update compatibility statements of the latest stable + in-development version, to add that new OpenJDK version, assuming we were already testing against the EA -- which we generally do.
* Upon versions of OpenJDK becoming unsupported, we should update compatibility statements of the in-development version *only* to remove that older OpenJDK version.
Latest stable shouldn't change incompatibly, so shouldn't be updated.

There might be exceptions, such as keeping compatibility with unsupported OpenJDK versions due to downstream demand, but I think the above makes sense as a general rule.